lib/gitlab/markdown.rb

```diff
@@ -91,6 +97,8 @@ def filters
       [
         Gitlab::Markdown::SanitizationFilter,
 
+        Gitlab::Markdown::RelativeLinkFilter,
+
```

Comment:
You can remove this newline and have it grouped with the Emoji/ToC/Autolink filters.